src/org/whstsa/library/util/DayGenerator.java

```diff
@@ -3,15 +3,14 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Random;
+import java.util.UUID;
 import java.util.stream.Collectors;
 
 import org.whstsa.library.Tester;
 import org.whstsa.library.api.BookType;
 import org.whstsa.library.api.IPerson;
 import org.whstsa.library.api.books.IBook;
-import org.whstsa.library.api.exceptions.CannotDeregisterException;
-import org.whstsa.library.api.exceptions.CheckedInException;
-import org.whstsa.library.api.exceptions.NotEnoughMoneyException;
+import org.whstsa.library.api.exceptions.*;
```

Comment:
Don't do wildcard imports unless you are going to use every (or most) of the classes in the package. This is important for memory management.